.github/workflows/spread.yaml

```
@@ -11,8 +11,10 @@ jobs:
         runner:
           - name: X64
             runs-on: ubuntu-latest
-          - name: ARM64
-            runs-on: [noble, ARM64, large]
+          # Execution of arm64 github runners on forked branches was disabled due to a recently found attack vector.
+          # see https://warthogs.atlassian.net/browse/ROCKS-1890
```

```suggestion
          # Execution of arm64 github runners on forked branches was disabled
          # due to a recently found attack vector.
          # TODO re-enable this runner once the above mentioned issue is fixed.
```

Let's not paste JIRA links inside the source, as they are not public.